Products/ZenUI3/browser/device_export.py

```diff
@@ -82,18 +85,7 @@ def csv(response, devices, fields):
                 if isinstance(value, list):
                     value = "|".join([v.get('name') for v in value])
                 if isinstance(value, dict):
-                    if field == 'events':
-                        for event_type in value.keys():
-                            fields_to_write.append("%s_events_count" % event_type)
-                            fields_to_write.append("%s_events_acknowledged" % event_type)
-                            data.append(str(value.get(event_type).get('count')))
-                            data.append(str(value.get(event_type).get('acknowledged_count')))
-                    else:
-                        value = value.get('name', None)
-                if not field == 'events':
-                    data.append(str(value).strip() if value or value is 0 else '')
-            if not wrote_header:
-                writer.writerow(fields_to_write)
-                wrote_header = True
+                    value = event(value) if field == 'events' else value.get('name', None)
+                data.append(str(value).strip() if value or value is 0 else '')
```

Comment:
When there are multiple checks in an `if` expression, put parenthesis around the test, e.g.
`str(value).strip() if (value or value is 0) else ''`.
This improves readability.